vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -13,6 +13,14 @@
 class FP8Config(QuantizationConfig):
     """Config class for FP8."""
 
+    config_file_optional = True
```

Comment:
I'd prefer making it a class method like others

```python
class QuantizationConfig(ABC):
    @classmethod # Not an abstract method and returns True by default.
    def require_config_file(cls) -> bool:
        return True
```

So that we can do

```python
if not quant_cls.require_config_file() and not quant_config_files:
        return quant_cls()
```

Reply:
SGTM!

Reply:
Fixed, this was indeed a great suggestion. Much more elegant.